cni/network/network.go

```diff
@@ -622,6 +624,12 @@ func (plugin *NetPlugin) createNetworkInternal(
 		return nwInfo, fmt.Errorf("Failed to ParseCIDR for pod subnet prefix: %w", err)
 	}
 
+	var podSubnetV6Prefix *net.IPNet
+	_, podSubnetV6Prefix, err = net.ParseCIDR(ipamAddResult.ipv6Result.IPs[0].Address.String())
```

Comment:
I'm curious, would this always be populated?
In ipv4 only mode, there would be no ipv6result, so there would be no IPs, and I think indexing by zero into the array would cause a panic?

Reply:
So maybe like Jaeryn is saying we only parse the IPv6 and add it to the nwinfo if it's ipv6